scan/cli.go

```diff
@@ -216,6 +241,79 @@ func ScanCmd(c *cli.Context) error {
 	return commands.Exec(scanCmd)
 }
 
+// New Build-Scan command that works directly with Xray
```

Comment:
// New Build-Scan command that works directly with Xray
-->
// Scan published builds with Xray